HW15/HW15_3.py

```diff
@@ -0,0 +1,49 @@
+CHANNELS = ["Discovery", "BBC", "TV1000"]
+DICT_CHANNELS = dict(enumerate(CHANNELS, start=1))
+CHENNEL = DICT_CHANNELS.get(1)
+CHENNEL_NUMBER = 1
+
+
+class ConrollerTV:
+    def __init__(self, channels=list) -> None:
+        pass
+
+    def first_channel():
+        global CHENNEL, CHENNEL_NUMBER
```

Comment:
I would recommend to refrain from using `global`. You could assign channel and channel number (perhaps only number because you can get a channel itself using a number, therefore, no need to use two variables) as a `self` attribute in init. This way you'd be able to preserve state of each object (instead of only a single one). I general, usage of `global` is discouraged.